src/schema/v2/CollectorProfile/helpers/formatName.ts

```diff
@@ -0,0 +1,33 @@
+type NameFormatType = "default" | "public"
+
+const formatName = (name: string, format: NameFormatType): string => {
+  switch (format) {
+    case "public":
+      return formatPublicName(name)
+    case "default":
+    default:
+      return name
+  }
+}
+
+export const formatPublicName = (name: string): string => {
+  if (!name || typeof name !== "string") {
+    throw new Error("Invalid name.")
+  }
+
+  const nameParts = name.split(" ")
+
+  if (nameParts.length === 1) {
+    return nameParts[0]
+  } else if (nameParts.length > 1) {
+    const firstName = nameParts[0]
+    const lastName = nameParts.pop()
+    const lastNameInitial = lastName ? lastName.charAt(0) : ""
+
+    return `${firstName} ${lastNameInitial}.`.trim()
```

Comment:
But this change is already going to introduce maybe an unwanted dot to where we are using it. I will leave it for your judgment to see if we remove or not because I don't know besides volt-v2 where else we're using this field.

Reply:
<img width="300" alt="image" src="https://github.com/artsy/metaphysics/assets/15792853/88a1395d-5177-4a52-a626-30d0f03de859">

here we'd show instead of `Jenna Poczic`, `Jenna.`, right? That's design changes, if Design is fine with it then let's move on.

Reply:
oh my bad, that's what I mean! if we're okay with changing the existing designs is okay from my side 👍